<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" ignored="0" total="3" passed="1">
  <reporter-output>
    <line>
      <![CDATA[Sucessfully login]]>
    </line>
    <line>
      <![CDATA[Not login]]>
    </line>
    <line>
      <![CDATA[Not login]]>
    </line>
  </reporter-output>
  <suite name="Default suite" duration-ms="106161" started-at="2020-03-31T15:13:13Z" finished-at="2020-03-31T15:14:59Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="106161" started-at="2020-03-31T15:13:13Z" finished-at="2020-03-31T15:14:59Z">
      <class name="com.ivavsys.TestPOM.LoginTest">
        <test-method status="PASS" signature="beforeTest()[pri:0, instance:com.ivavsys.TestPOM.LoginTest@1fbdac4]" name="beforeTest" is-config="true" duration-ms="13994" started-at="2020-03-31T15:13:13Z" finished-at="2020-03-31T15:13:27Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method status="PASS" signature="login(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.ivavsys.TestPOM.LoginTest@1fbdac4]" name="login" duration-ms="2629" started-at="2020-03-31T15:13:27Z" data-provider="getData" finished-at="2020-03-31T15:13:30Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Admin]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Admin123]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Home]]>
              </value>
            </param>
          </params>
          <reporter-output>
            <line>
              <![CDATA[Sucessfully login]]>
            </line>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method status="PASS" signature="afterTest(org.testng.ITestResult)[pri:0, instance:com.ivavsys.TestPOM.LoginTest@1fbdac4]" name="afterTest" is-config="true" duration-ms="95" started-at="2020-03-31T15:13:30Z" finished-at="2020-03-31T15:13:30Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=login status=SUCCESS method=LoginTest.login(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.ivavsys.TestPOM.LoginTest@1fbdac4] output=Sucessfully login]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
        <test-method status="PASS" signature="beforeTest()[pri:0, instance:com.ivavsys.TestPOM.LoginTest@1fbdac4]" name="beforeTest" is-config="true" duration-ms="12302" started-at="2020-03-31T15:13:30Z" finished-at="2020-03-31T15:13:42Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method status="FAIL" signature="login(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.ivavsys.TestPOM.LoginTest@1fbdac4]" name="login" duration-ms="31802" started-at="2020-03-31T15:13:42Z" data-provider="getData" finished-at="2020-03-31T15:14:14Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Admin]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[123.0]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Home]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.fail(Assert.java:103)
at com.ivavsys.pom.EnterTimeTrack.Vaildate(EnterTimeTrack.java:23)
at com.ivavsys.TestPOM.LoginTest.login(LoginTest.java:32)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[Not login]]>
            </line>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method status="PASS" signature="afterTest(org.testng.ITestResult)[pri:0, instance:com.ivavsys.TestPOM.LoginTest@1fbdac4]" name="afterTest" is-config="true" duration-ms="591" started-at="2020-03-31T15:14:14Z" finished-at="2020-03-31T15:14:15Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=login status=FAILURE method=LoginTest.login(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.ivavsys.TestPOM.LoginTest@1fbdac4] output=Not login]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
        <test-method status="PASS" signature="beforeTest()[pri:0, instance:com.ivavsys.TestPOM.LoginTest@1fbdac4]" name="beforeTest" is-config="true" duration-ms="12307" started-at="2020-03-31T15:14:15Z" finished-at="2020-03-31T15:14:27Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method status="FAIL" signature="login(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.ivavsys.TestPOM.LoginTest@1fbdac4]" name="login" duration-ms="31780" started-at="2020-03-31T15:14:27Z" data-provider="getData" finished-at="2020-03-31T15:14:59Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[UserName]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Admin123]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Home]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.fail(Assert.java:103)
at com.ivavsys.pom.EnterTimeTrack.Vaildate(EnterTimeTrack.java:23)
at com.ivavsys.TestPOM.LoginTest.login(LoginTest.java:32)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[Not login]]>
            </line>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method status="PASS" signature="afterTest(org.testng.ITestResult)[pri:0, instance:com.ivavsys.TestPOM.LoginTest@1fbdac4]" name="afterTest" is-config="true" duration-ms="519" started-at="2020-03-31T15:14:59Z" finished-at="2020-03-31T15:14:59Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=login status=FAILURE method=LoginTest.login(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.ivavsys.TestPOM.LoginTest@1fbdac4] output=Not login]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
      </class> <!-- com.ivavsys.TestPOM.LoginTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
